src/cmap/connection.ts

```diff
@@ -165,6 +168,7 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
   public delayedTimeoutId: NodeJS.Timeout | null = null;
   public generation: number;
   public readonly description: Readonly<StreamDescription>;
+  public established: boolean;
```

Comment:
We discussed leaving a TSDoc comment here, we should include what it means and "who" sets it. I think the comment inside `connect()` mostly fits well here.

Reply:
Sounds good, I've updated the TSDoc now